src/hoc/withTheme.js

```diff
@@ -57,5 +39,11 @@ export default (Component: ComponentType<any>) => {
     }
   }
 
-  return hoistStatics(WithTheme, Component)
+  const ForwardRef = React.forwardRef((props, ref) => (
+    <WithTheme {...props} forwardedRef={ref} />
```

Comment:
Couldn't we call this `innerRef`, like we did before? Then we wouldn't even have to change the above code, and old code would keep working as-is I think.

Reply:
I guess, but with `forwardedClass` also being added I like keeping the recommended name for similarity

Reply:
Not sure that's worth much, given it's only internal? I'd prefer if the migration path was nicer, especially if we can warn in dev mode if `innerRef` was explicitly passed and warn to change to `ref`.

Reply:
Well the concept of `innerRef` is totally gone now in general. Just use `ref`. Very easy codemod or grep+replace

Reply:
I'll add a dev warning for innerRef though